### PR TITLE
[regression test](routine test) add case for desired_concurrent_number

### PR DESCRIPTION
**Pull request by liugddx**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run P0

**Comment by liugddx** [comment]
run pipelinex_P0

**Comment by liugddx** [comment]
run pipelinex_p0

**Comment by liugddx** [comment]
run p0

**Comment by liugddx** [comment]
run pipelinex_p0

**Comment by liugddx** [comment]
run buildall

**Comment by liugddx** [comment]
run pipelinex_p0

**Comment by liugddx** [comment]
run p0

**Comment by liugddx** [comment]
run pipelinex_p0

**Comment by liugddx** [comment]
run pipelinex_p0

File: regression-test/suites/load_p0/routine_load/test_routine_load.groovy [hunk 1]
```diff
@@ -233,6 +233,169 @@ suite("test_routine_load","p0") {
...
+                sql new File("""${context.file.parent}/ddl/${tableName}_create.sql""").text
+
+                def name = "routine_load_" + tableName
+                sql """
+                    CREATE ROUTINE LOAD ${jobs[i]} ON ${name}
+                    COLUMNS(${columns[i]}),
+                    COLUMNS TERMINATED BY "|"
+                    PROPERTIES
+                    (
+                        "desired_concurrent_number" = "3",
```
Comment (HHoflittlefish777) [inline regression-test/suites/load_p0/routine_load/test_routine_load.groovy:251]: need check desired_concurrent_number value

**Comment by liugddx** [comment]
run buildall

**Review by sollhui — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**